src/exec/src/bin/occlum_exec_server.rs

```diff
@@ -61,8 +63,15 @@ fn main() {
         DEFAULT_SOCK_FILE,
     );
 
+    let mut hasher = DefaultHasher::new();
+    sockfile.hash(&mut hasher);
+
+    let mut sockfile = String::from("/tmp/occlum_");
+    sockfile.push_str(hasher.finish().to_string().as_str());
+    let sockfile = &*sockfile;
```

Comment:
String has already implemented `Deref<Target=str>`. I guess we do not necessarily need this here and can directly use check_server_status(&sockfile) instead.
Same above for the occlum_exec_client.rs.

Reply:
line 71 is used to change sockfile to be immutable. Just make it more clear to myself.

Reply:
To be honest, I am quite confused with `sockfile`: four variables named `sockfile` is defined at line 59, 60, 69, and 71, respectively. The content and meaning of sockfile keeps changing (rapidly), which is a bad sign for readability and correctness. I think they really should have different names to make their differences and relationship more clear and straightforward. The client code has the same issue.

Reply:
updated